src/widget/form/settings/avform.cpp

```diff
@@ -11,6 +11,7 @@
 #include <QDebug>
 #include <QScreen>
 #include <QShowEvent>
+#include <ranges>
```

Comment:
Is qTox C++20? (`std::ranges::reverse_view`)

Reply:
https://github.com/TokTok/qTox/blob/master/CMakeLists.txt#L89 yes